lib/components/user/nav-login-button.css

```diff
@@ -25,6 +25,11 @@
   background-color: #080808;
 }
 
+li.header {
```

Comment:
Flagging that if we make this into an actual header, this css will have to change